packages/atlas/src/views/viewer/DiscoverView/DiscoverView.tsx

```diff
@@ -18,8 +19,8 @@ import {
 } from './DiscoverView.styles'
 
 export const DiscoverView: FC = () => {
-  const { categories, totalVideosCount, loading } = useCategories()
-  const mappedVideoCategories = useCategoriesMatch()
+  const { categories, totalVideosCount, loading } = useQnCategories()
+  const videoCategories = useVideoCategoriesWithCounter()
```

Comment:
This should be called `displayCategories`. In this component we have both `categories` and `videoCategories`, that's super misleading

Reply:
Btw, why do we even need QN categories here? I would expect that display categories will be enough?